translations/en.json

```diff
@@ -0,0 +1,1215 @@
+    "notANumber": "{description} must be a number.",
+    "notAnInteger": "{description} must be an integer.",
+    "greaterThan": "{description} must be greater than {gt}.",
+    "greaterThanOrEqualTo": "{description} must be greater than or equal to {gte}.",
+    "equalTo": "{description} must be equal to {is}.",
+    "lessThan": "{description} must be less than {lt}.",
+    "lessThanOrEqualTo": "{description} must be less than or equal to {lte}.",
+    "otherThan": "{description} must be other than {value}.",
+    "odd": "{description} must be odd.",
+    "even": "{description} must be even.",
+    "positive": "{description} must be positive.",
+    "date": "{description} must be a valid date.",
+    "onOrAfter": "{description} must be on or after {onOrAfter}.",
+    "onOrBefore": "{description} must be on or before {onOrBefore}.",
+    "email": "{description} must be a valid email address.",
+    "phone": "{description} must be a valid phone number.",
+    "url": "{description} must be a valid url.",
+    "email_registered": "This email address has already been registered.",
+    "email_invalid": "Invalid email address. If this should not have occurred, please report this to {supportEmail}",
+    "email_match": "Email addresses must match.",
+    "password_email": "Your password cannot be the same as your email address.",
+    "password_old": "Your new password cannot be the same as your old password.",
+    "password_match": "Passwords must match.",
+    "recaptcha": "Please complete reCAPTCHA.",
+    "affirm_terms": "You must read and agree to the Terms of Use and Privacy Policy.",
+    "min_subjects": "You must select at least {minLength} subject(s).",
+    "node_license_invalid": "Invalid required fields for the license",
+    "node_license_missing_fields": "The following required fields are missing: {missingFields}"
+  },
+  "validated_input_form": {
+    "discard_changes": "Discard changes"
+  },
+  "node_navbar": {
+    "toggle": "Toggle navigation",
+    "project_nav": "Project Navigation",
+    "wiki": "Wiki",
+    "analytics": "Analytics",
+    "registrations": "Registrations",
+    "files": "Files",
+    "contributors": "Contributors",
+    "addons": "Add-ons",
+    "settings": "Settings",
+    "comments": "Comments"
+  },
+  "status": {
+    "welcome_message": "<h1>Welcome to OSF!</h1><p>Visit our <a href=\"http://help.osf.io/\" target=\"_blank\" rel=\"noreferrer\">Guides</a> to learn about creating a project, or get inspiration from <a href=\"https://osf.io/explore/activity/#popularPublicProjects\">popular public projects</a>.</p>",
+    "alternate_email_error": "The email address has <b>NOT</b> been added to your account. Please log out and revisit the link in your email. Thank you.",
+    "remove_addon": "Because the GitHub add-on for {extra.category} \"{extra.title}\" was authenticated by {extra.user}, authentication information has been deleted.",
```

Comment:
Nested keys like `{extra.category}` do not work, so the information needs to be passed to this differently. I think that may involve some back-end changes with the `push_status_message` code.

Reply:
We'd just have to break it up and send the pieces to `t`.